src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1147,6 +1147,8 @@ Color.white, new Procedure("condition4"),
 			plant.dropAmount = 4;
 			plant.xpAmountMin = 2;
 			plant.xpAmountMax = 5;
+			plant.blockBounciness = new NumberProcedure(emptyLists ? null : "number1", 0);

```

Comment:
0 is the default value. Tests should not use default values so they can verify if default value changed